index.html

```diff
@@ -47,6 +47,167 @@ <h1 class="intro">Hey there. I’m Tobin <br /><span>I’m a software<br/>develo
             </li>
         </ul>
     </section>
+    <section  class="work-section">
+        <div class="disable-btn">
+            <img src="IMG/Disabled.png" alt="down-btn">
+        </div>
+        <div class="section-title">
+            <h2>My Recent Works</h2>
+        </div>
+        <div class="green-line">
+            <img src="IMG/Indicator.png" alt="green line">
+        </div>
+        <div class="project-container">
+            <div class="project-card">
+                <img src="IMG/Plaholder.png" alt="Project image">
+                <div class="card-title">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                </div>
+                <ul class="project-technologies">
+                    <li>Ruby on rails</li>
+                    <li>JavaScript</li>
+                    <li>css</li>
+                    <li>html</li>
+                </ul>
+                <div class="project-btn">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="project-card">
+                <img src="IMG/Plaholder.png" alt="Project image">
+                <div class="card-title">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                </div>
+                <ul class="project-technologies">
```

Comment:
## [OPTIONAL]

- [ ] I think it would be nice to improve the work section code by putting all projects in a `<ul>` tag.
- Your implementation
![Screenshot from 2021-11-27 15-50-49](https://user-images.githubusercontent.com/65068771/143684880-0640b7b6-42de-402b-b288-42eb2eaeb70d.png)
- A better implementation 
```
 <ul class="project-container">
      <li class="project-card"></li>
      <li class="project-card"></li>
      <li class="project-card"></li>
      <li class="project-card"></li>
      <li class="project-card"></li>
</ul>
```